components/feature/intent/src/main/java/mozilla/components/feature/intent/processing/TabIntentProcessor.kt

```diff
@@ -68,10 +70,11 @@ class TabIntentProcessor(
             false
         } else {
             val url = WebURLFinder(extraText).bestWebURL()
+            val source = SessionState.Source.External.ActionView(intent.externalPackage())
```

Comment:
This should be `External.ActionSend`. Can we cover this in TabIntentProcessorTest?

Reply:
Fixed and added test coverage for this. Thanks for the catch!